e2e/external_dep/BUILD

```diff
@@ -38,3 +38,26 @@ build_test(
         "lib.ts56rc",
     ],
 )
+
+ts_project(
+    name = "external-deps",
+    srcs = ["lib.ts"],
+    composite = True,
+    out_dir = "external-deps",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@sub_external//:lib1",
+        "@sub_external//:tsconfig-dict",
```

Comment:
Maybe add the 2 `subdir` ones here as well? Just to test they still work as `ts_project(deps)` even though the code doesn't load them 🤷).

Reply:
done